### ISSUE TITLE
Allow progress texture to display stacks or dynamic info other than duration

### ISSUE THREAD

**Issue by Stanzilla** [WeakAuras/WeakAuras2#1445]
Moved from https://www.wowace.com/projects/weakauras-2/issues/440

**Is your feature request related to a problem? Please describe.**
I would like to see a feature where progress textures are able to use dynamic information other than duration.

The use case I'm specifically thinking of is Earth Shield charges. I'd like to use a progress texture to show graphically how many charges are left on my active earth shield using an Aura trigger on the Group unit. Potentially could be useful for things like shields if that info is available.

Right now I'm looking at doing a custom trigger for it but if it were implemented as a standard feature it'd be able to be used with multi targets to track multiple of whatever is being tracked (e.g. all of the Earth Shields in the raid instead of only mine).

**Describe the solution you'd like**
In case my explanation wasn't very good, I've provided a sample WeakAuras string
```
dWIbcaGlPuBdqzMsHMRu1SbDBkzNsL9s2nk7xG6Nsjdta8BunyPGHlkhuaDmr1cfGwkIklxYtvTmbYZryQinzkmDOlcWLP66cTzkA7aYhru1NfXRfKrIO0JbA0uQXlOojICyfNwPZlf9xrATaQMgIIvUO6Rojg6L5a69Ka5mI(N5qyA3GXYtFOLXnbmfARtOhu782bODoW0T0n0nevhKd5LOc1XAts8suDcCadLUmtUoipYq9iBr9KIhZ0zJLNUmtU(eRrNnwoHOQlxhWuODWnC71HgcfQd5JHO6gRP5cgHytr1TIqCfvOq9IdDr1TIqCfvOqDCGodfv3kcXvuHc1Rb0fv3kcXvuHcfQBYlpP4rMb1dJXY1PEymwor)ZCimqqCbAWLZAbKxWPjrFahWqWHWoL8L7m9YtwoJ2mZ1ZkEs5eeOhgJLRVg6acpwzqhWuO9TxhAOZIw0Ats8IqD568ADgo4Yz6KTf5ib8wD0aBuO(iQoGPq7GB42Rdn0jXqVmhqVNeiNr0)mhct7gmwE6dTmUjGPqBH6aPUCYeuUqja
```
Basically I'd like this progress texture to count down my stacks instead of the time remaining.

I guess what I was really trying to achieve here is two things:

1) To be able to customise the duration info without having to write a custom trigger and untrigger to detect the buff (WeakAuras already has excellent support for this and it feels like reinventing the wheel to try and rewrite it). I personally would be fine with writing custom code to do the duration info if the owner of the buff were already identified. 2) To be able to use customised duration info with the multi-target feature.

If max stacks is the issue, the API doesn't know what the max stacks are but the user usually does. They could provide it as input.

**Describe alternatives you've considered**

**Additional context**

**Comment by InfusOnWoW** [comment]
Duplicate of https://github.com/WeakAuras/WeakAuras2/issues/1449.

Or rather my half-finished implementation solves both at the same time.
